docs/deployment/feature_gates.md

```diff
@@ -62,6 +60,9 @@ The following tables are a summary of the feature gates that you can set on diff
 | DisallowKubeconfigRotationForShootInDeletion |         | `Removed` | `1.38` |        |
 | Logging                                      | `false` | `Alpha`   | `0.13` | `1.40` |
 | Logging                                      | `false` | `Removed` | `1.41` |        |
+| AdminKubeconfigRequest                       | `false` | `Alpha`   | `1.24` | `1.38` |
+| AdminKubeconfigRequest                       | `true`  | `Beta`    | `1.39` | `1.41` |
+| AdminKubeconfigRequest                       | `true`  | `GA`      | `1.42` |        |
```

Comment:
Looks like there is 1 more occurrence that has to be adapted:

https://github.com/gardener/gardener/blob/ab23368095481b5be14aaebdece6b65214df1f1e/example/gardener-local/controlplane/values.yaml#L159